packages/calcite-components/src/utils/floating-ui.ts

```diff
@@ -433,10 +456,17 @@ export async function reposition(
   options: Parameters<typeof positionFloatingUI>[1],
   delayed = false,
 ): Promise<void> {
-  if (!component.open || !options.floatingEl || !options.referenceEl) {
+  if (!options.floatingEl || !options.referenceEl) {
     return;
   }
 
+  if (!component.open) {
+    Object.assign(options.floatingEl.style, hiddenFloatingElStyle);
+    return;
+  }
+
+  Object.assign(options.floatingEl.style, initialFloatingElStyle);
```

Comment:
We assign the initial floating el styles before any positioning happens. This means the top and left are both zero and the position is absolute. The width and height of zero is removed.